website/versioned_docs/version-v1.x/cli.md

```diff
@@ -29,14 +29,14 @@ pnpm run hardhat
+| `deployPoll`         | Deploy a new poll                                                        | `-t, --duration <pollDuration>`: The poll duration <br/> `-i, --int-state-tree-depth <intStateTreeDepth>`: The int state tree depth <br/> `-b, --msg-batch-depth <messageTreeSubDepth>`: The message tree sub depth <br/> `-m, --msg-tree-depth <messageTreeDepth>`: The message tree depth <br/> `-v, --vote-option-tree-depth <voteOptionTreeDepth>`: The vote option tree depth <br/> `-pk, --pubkey <coordinatorPubkey>`: The coordinator public key <br/> `-x, --maci-address <maciAddress>`: The MACI contract address <br/> `-q, --quiet`: Whether to print values to the console                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
I found there's a option called `"-vk, --vkRegistryAddress <vkRegistryAddress>", "the vk registry contract address"` in the cli/index.ts file